tests/introspection/tests.py

```diff
@@ -170,6 +170,9 @@ def test_get_primary_key_column(self):
     def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
+        if connection.vendor == 'postgresql':
```

Comment:
This should be a database feature flag rather than a vendor check. It would be great to investigate what other database backends provide this functionality and to also implement it there.